ui/src/testing/stubs/cohorts-api-stub.ts

```diff
@@ -0,0 +1,142 @@
+import {convertToResources, ResourceType} from 'app/utils/resourceActionsReact';
+import {
+  Cohort,
+  CohortsApi,
+  EmptyResponse,
+  RecentResource,
+  Workspace,
+  WorkspaceAccessLevel
+} from 'generated/fetch';
+import {CohortListResponse} from 'generated/fetch/api';
+import {WorkspaceStubVariables} from './workspace-service-stub';
+
+export let DEFAULT_COHORT_ID = 1;
+export let DEFAULT_COHORT_ID_2 = 2;
+
+class CohortStub implements Cohort {
+  id?: number;
+
+  name: string;
+
+  criteria: string;
+
+  type: string;
+
+  description?: string;
+
+  creator?: string;
+
+  creationTime?: number;
+
+  lastModifiedTime?: number;
+
+  workspaceId: string;
+
+  constructor(cohort: Cohort, wsid: string) {
+    this.creationTime = cohort.creationTime;
+    this.creator = cohort.creator;
+    this.criteria = cohort.criteria;
+    this.description = cohort.description;
+    this.id = cohort.id;
+    this.lastModifiedTime = cohort.lastModifiedTime;
+    this.name = cohort.name;
+    this.type = cohort.type;
+    this.workspaceId = wsid;
+  }
+}
+
+export class CohortsApiStub extends CohortsApi {
+  public workspaces: Workspace[];
+  public cohorts: CohortStub[];
+  public resourceList: RecentResource[];
```

Comment:
I don't this stub should have `workspaces` or a `resourceList`, they also aren't used anywhere AFAICT.

Reply:
Not addressed